.gitattributes

```diff
@@ -1 +1 @@
-*  text=auto
\ No newline at end of file
+  *  text=auto eol=lf
```

Comment:
This shouldn't be kept in this PR.
And if you have the correct git settings, everything should work fine without touching this file. Can you confirm?

Reply:
indeed